docker.yml

```diff
@@ -14,16 +14,25 @@ resource:
       tags:
         Name: '${var.koding_user_username}-${var.koding_group_slug}'
       user_data: |-
+        #!/bin/bash
+        
+        set -eo pipefail
+        set -x
+        
         export DEBIAN_FRONTEND=noninteractive
-        #setting the new source
         
-        #Updating packages
-        apt-get update -y
-        #Install Docker engine
-        sudo apt-get install apt-transport-https ca-certificates
-        sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
-        echo 'deb https://apt.dockerproject.org/repo ubuntu-trusty main' > /etc/apt/sources.list.d/docker.list
-        sudo apt-get update
-        sudo apt-get purge -y lxc-docker
-        sudo apt-get install -y linux-image-extra-$(uname -r)
-        sudo apt-get install -y --force-yes docker-engine
+        apt-get update && apt-get install -y \
```

Comment:
And operator is redundant because `errexit` will cause an early exit if a failure occurs.